awsaws-ses/aws-ses/src/main/java/com/brother/awsses/dto/EmailInfo.java

```diff
@@ -0,0 +1,13 @@
+package com.brother.awsses.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+public class EmailInfo {
+    private String[] receiverAddresses;
```

Comment:
we can rename **receiverAddresses** to **toAddresses** for matching with below fields.